go.sum

```diff
@@ -231,6 +231,8 @@ github.com/google/cadvisor v0.37.0 h1:t3txV4zNZZGTuwuA/Onm3HToPhg16GjigAHZHEVIz+
 github.com/google/cadvisor v0.37.0/go.mod h1:OhDE+goNVel0eGY8mR7Ifq1QUI1in5vJBIgIpcajK/I=
 github.com/google/go-cmp v0.4.0 h1:xsAVV57WRhGj6kEIi8ReJzQlHHqcBYCElAvkovg3B/4=
 github.com/google/go-cmp v0.4.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
+github.com/google/go-flow-levee v0.1.3 h1:bg7F/zMjwFzWrV/pF4TIfR2cO+K+zdpG1k9eWgFp3WM=
+github.com/google/go-flow-levee v0.1.3/go.mod h1:C435ZDtvbgiwHqcjW2tAPJseY4bpiLJrarS3il+uNek=
```

Comment:
this should not be in the primary vendor / go.sum
this is a development-time dependency only?

we have a pattern for avoiding this with the hack/verify* targets. see e.g. staticcheck

Reply:
/hold

Reply:
this hold was addressed @PurelyApplied -- btw ~anyone can `/hold cancel` :sweat_smile: